### ISSUE TITLE
Error with jekyll 3.9.0: missing dependency kramdown-parser-gfm

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma.com#320]
According to 3.9.0 https://jekyllrb.com/news/2020/08/05/jekyll-3-9-0-released/ this gem should be installed. Perhaps this is a [jekyll-theme-open-project](https://github.com/riboseinc/jekyll-theme-open-project) dependency?

```
  Dependency Error: Yikes! It looks like you don't have kramdown-parser-gfm or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- kramdown-parser-gfm' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
  Liquid Exception: kramdown-parser-gfm in /_layouts/default.html
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    kramdown-parser-gfm
```

**Comment by ronaldtse** [comment]
ping @strogonoff

**Comment by codepug** [comment]
Adding the following to Gemfile seemed to fix it for me:

gem "kramdown-parser-gfm"

**Comment by strogonoff** [comment]
Yeah, that’s Jekyll installation, which is outside of plugin’s scope. Jekyll is supposed to be installed per official instructions [which we link to](https://github.com/riboseinc/jekyll-theme-open-project#getting-started-with-ruby-and-jekyll). I don’t think it makes sense for a plugin to duplicate Jekyll installation/migration instructions. We only specify what’s needed _on top of_ Jekyll’s own instructions.

**Comment by strogonoff** [comment]
By the way, jekyll-theme-open-project should work with Jekyll 4 I believe. Documentation though still mentions 3, which I should update while it’s still maintained…

**Comment by ronaldtse** [comment]
This was actually fixed in 40c62e51 back in August 2020. Closing.

**Comment by ronaldtse** [comment]
@strogonoff Now I remember -- the point of this ticket was to tell you that jekyll-theme-open-project depends on kramdown-parser-gfm, and therefore its gemspec should be updated.
